### ISSUE TITLE
Write a tribble codec for parsing Picard interval_list format

### ISSUE THREAD

**Issue by droazen** [broadinstitute/gatk#5788]
As requested by @yfarjoun

**Comment by droazen** [comment]
@yfarjoun I heard you might be working on this?

**Comment by yfarjoun** [comment]
yeah...in htsjdk thou

**Comment by droazen** [comment]
@yfarjoun Right, the intention of this ticket was to implement the codec in htsjdk, then add a GATK integration test proving that we can now access interval_list files as tribble features.

**Comment by yfarjoun** [comment]
part 1 is here: https://github.com/samtools/htsjdk/pull/1327
